docs/admin/kubernetes-ops/README.md

```diff
@@ -7,7 +7,14 @@ tags:
 
 # Overview
 
-Please use the left navigation menu to browse the content of this section while we are still working on developing content for `Overview` page.
+The Kubernetes Operation Guide helps you learn about the common operations for Janssen installation on Kubernetes.
 
```

Comment:
Provide more information. Common operations, a reference to where the installation is at. A general walk of the arch and how the containers start. I would also add an arch diagram. Look at this https://gluu.org/docs/gluu-server/4.4/img/kubernetes/cn-general-arch-diagram.svg . You can import this into draw.io and modified it so it fits